packages/flutter/lib/src/rendering/object.dart

```diff
@@ -3424,7 +3429,7 @@ abstract class _SemanticsFragment {
 
   /// Returns [_InterestingSemanticsFragment] describing the actual semantic
   /// information that this fragment wants to add to the parent.
-  Iterable<_InterestingSemanticsFragment> get interestingFragments;
```

Comment:
is there necessary?

Reply:
A more specific type is generally better, in this case it is a private API so there should be no concerns.